applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -62,7 +62,7 @@ async fn main() -> Result<(), MmProxyError> {
     let service = make_service_fn(|_conn| future::ready(Result::<_, Infallible>::Ok(xmrig_service.clone())));
 
     println!("\nListening on {}...\n", addr);
-    Server::bind(&addr).serve(service).await?;
+    Server::try_bind(&addr)?.serve(service).await?;
```

Comment:
```suggestion
    match Server::try_bind(&addr) {
        Ok(builder) => builder.serve(service).await?,
        Err(_) => {
            println!(
                "\nWarn: Cannot bind to port in '{}', it may be part of a Port Exclusion Range. Please \n      try to \
                 use another port for the 'proxy_host_address' in 'config/config.toml' and for the \n      applicable \
                 XMRig '[pools][url]' or '[pools][self-select]' config setting that can be found \n      in \
                 'config/xmrig_config_***.json' or '<xmrig folder>/config.json'.\n",
                addr
            );
            Server::try_bind(&addr)?;
        },
    }
```

Reply:
Thanks - changes pushed